layouts/_default/newsletters-archive.html

```diff
@@ -25,40 +29,40 @@
 
   {{ $titleClass := cond (gt (countwords .Title) 2) "is-size-1" "is-size-0" }}
   {{ $subheadClass := cond (gt (countwords $subtitle) 15) "is-size-4" "is-size-3" }}
-  <section
-    class="section columns is-centered is-multiline"
-    data-ga-category="list"
-  >
+  <section class="bg-white px-4 pt-12 pb-2 md:pb-6" data-ga-category="list">
     {{ if not $p.HasPrev }}
-      <div class="column is-three-fifths">
-        <header class="mb-5">
+      <div class="mx-auto w-full max-w-screen-lg">
+        <header class="mb-5 mx-auto">
           <h1
-            class="title has-text-centered {{ $titleClass }} is-size-2-touch is-spaced"
+            class="text-center font-sans text-3xl mb-6 sm:text-4xl md:text-6xl font-bold leading-none tracking-wide {{ $titleClass }}"
           >
             {{ .Title }}
           </h1>
           <h2
-            class="subtitle has-text-centered {{ $subheadClass }} is-size-4-touch mb-6"
+            class="text-center font-sans text-2xl font-light mb-12 text-s-8 {{ $subheadClass }}"
           >
             {{ $subtitle }}
           </h2>
-
-          {{ partial "component/hero-image-block.html" . }}
+          <div class="max-w-prose m-auto">
+            {{ partial "tw/image-hero.html" . }}
+          </div>
         </header>
         {{ with .Content }}
-          <div class="content mb-6">
+          <div class="article-content max-w-prose mx-auto">
             {{ . }}
           </div>
         {{ end }}
       </div>
     {{ end }}
-    <div class="column is-four-fifths mb-6" id="archives">
-      <nav class="tile is-ancestor is-vertical">
+    <div class="column is-four-fifths mb-6 p-2 font-serif" id="archives">
+      <nav class="mb-16 mx-auto max-w-screen-lg">
         {{ range $pages.GroupByDate "January 2006" }}
           <div class="tile is-parent">
-            <h2 class="title">{{ .Key }}</h2>
+            <h2 class=" text-3xl font-extrabold font-raleway mt-16 mb-9">
+              {{ .Key }}
+            </h2>
           </div>
-          <div class="tile is-wrapped">
+          <div class="mt-4 grid gap-12 sm:grid-cols-2 md:grid-cols-3">
```

Comment:
Don't use the tile/list partial here. I want the tile stuff to be pure Bulma so we can drop the whole directory when Bulma is removed. You can use the tw/list-3-col-story.html partial instead. See how _default/list.html and _default/terms.html do it.

Reply:
This layout has choppy loading because it isn't getting the aspect ratio hint for images, I guess because it's relying on the Bulma aspect-ratio helper, so switching to list-3-col-story will be good for fixing that and getting the subgrid alignment automatically.